docs/administrators/metering.rst

```diff
@@ -61,11 +61,24 @@ Reported Metrics
 
 This section assumes familiarity with influxdb. See `here <https://docs.influxdata.com/influxdb/v1.7/concepts/key_concepts/#field-key>`_.
 
-All API metrics are reported under the measurement `metrics`.
+All metrics are reported under the measurement `metrics`.
+Different measurements are distinguished by a tag called `key`.
+
+Two main types of metrics are reported:
+1. Metrics related to API performance
+2. Others
+
+API performance metrics
++++++++++++++++++++++++
+Each API method is reported with a `key=rpc.{endpoint_name}`.
+The `endpoint_name` is the server's internal name for the endpoint.
+To know which url corresponds to which method, please consult either the `operationId`
+field of the `OpenAPI spec <./openapi.json>`_ or
```

Comment:
```suggestion
field of the `OpenAPI spec <./_specs/openapi.json>`_ or
```

Reply:
Did you test this? I think the leading dot will make this a relative path.